.coderabbit.yaml

```
@@ -1,201 +1,144 @@
+    on_push: true
+    on_pull_request: true
+    # 除外するファイル
+    exclude:
+      - "**/xcuserdata/**"
+      - "**/DerivedData/**"
+      - "**/Pods/**"
+      - "**/.build/**"
+      - "**/Preview Content/**"
+      - "**/*.xcassets/**"
+      - "**/fastlane/**"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)